com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -17,6 +17,31 @@ public sealed class NetworkObject : MonoBehaviour
         [SerializeField]
         internal uint GlobalObjectIdHash;
 
+        /// <summary>
+        /// Gets the Prefab Hash Id of this object if the object is registerd as a prefab otherwise it returns 0
+        /// </summary>
+        [HideInInspector]
+        public uint PrefabIdHash
+        {
+            get
+            {
+                foreach (var prefabList in NetworkManager.NetworkConfig.Prefabs.NetworkPrefabsLists)
+                {
+                    if (prefabList.Contains(gameObject))
+                    {
+                        return GlobalObjectIdHash;
+                    }
+                }
+
+                if ( NetworkManager.PrefabHandler.ContainsHandler( GlobalObjectIdHash ) )
```

Comment:
The standards check issues are due to the 2 whitespaces on line 36 position 21 and line 36 position 67. 
`if (NetworkManager.PrefabHandler.ContainsHandler( GlobalObjectIdHash ))`
👍